src/screens/BackupSheet.js

```diff
@@ -51,7 +53,10 @@ const BackupSheet = () => {
   const switchSheetContentTransitionRef = useRef();
   const { params } = useRoute();
   const dispatch = useDispatch();
-  const { selectedWallet, isWalletLoading } = useWallets();
+  const { selectedWallet, isWalletLoading, wallets } = useWallets();
+  const backupeableWalletsCount = wallets.filter(
```

Comment:
`backupeable` should be `backupable` hehe not sure if its even a word but its definitely more of a word without the `e`